docs/src/developer-guide.md

```diff
@@ -20,10 +20,10 @@ Building the Ray.jl project requires the following tools to be installed. This l
 4. Navigate to the root of the Ray.jl repo
 5. Install [Ray dependencies](https://docs.ray.io/en/latest/ray-contribute/development.html#preparing-to-build-ray-on-macos):
 
-```sh
-brew update
-brew install bazelisk wget
-```
+   ```sh
```

Comment:
are indented snippets just a personal preference?

Reply:
These were always intended to be a part of the list number above. I just forgot the syntax as it's odd to have a 3-space indentation.

Reply:
ah I see!